websites/G/Goodreads/presence.ts

```diff
@@ -0,0 +1,35 @@
+const presence = new Presence({

```

Comment:
```suggestion
const presence: Presence = new Presence({
```

TypeScript usually knows the `Presence` type, but better to be safe!

Reply:
no. dont even try to change it.